packages/coinstac-ui/app/render/components/user-account-controller.js

```diff
@@ -38,7 +37,7 @@ class UserAccountController extends Component {
 
   render() {
     return (
-      <UserAccount logout={this.logout} {...this.props} />
```

Comment:
Looks like `UserAccount` only needs a `user` prop, but that makes total sense. We might want to pass more stuff from `UserAccount Controller` to `UserAccount` in the future.